pydantic/fields.py

```diff
@@ -19,6 +19,8 @@
     cast,
 )
 
+from typing_extensions import Literal
```

Comment:
can we make `typing_extensions` an optional dependency by:
1. adding it as such in `setup.py`
2. catching import errors here and below.